draft-ietf-rtcweb-jsep.xml

```diff
@@ -3642,27 +3642,34 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         processing MUST stop and an error MUST be returned.
         <list style="symbols">
 
-          <t>If the PeerConnection state is "have-local-offer", and the
-          ICE ufrag or password changed from the previous remote
+          <t>If the ICE ufrag or password are unchanged from the previous remote description,
```

Comment:
The original structure actually didn't cover the "if the offer changed and the other side didn't" case, and the text just had it in leaf nodes. So I decided to put it upfront. Happy to talk about other structure.